src/components/reusable/errorBlock/errorBlock.stories.js

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit';
+import './index';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+import alertImg from './redfords/Alert_Redford.svg';
+import notFoundImg from './redfords/NotFound_Redford.svg';
+import stopImg from './redfords/Stop_Redford.svg';
+import timeoutImg from './redfords/Timeout_Redford.svg';
+
+export default {
+  title: 'Components/Error Block',
+  component: 'kyn-error',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/CQuDZEeLiuGiALvCWjAKlu/Applications---Component-Library?node-id=17386-106044&node-type=canvas&t=VYrUi3e41lGwsAHM-0',
+    },
+  },
+};
+
+const args = {
+  titleText: 'Error title',
+};
+
+export const ErrorBlock = {
+  args,
+  decorators: [
+    (story) => html`
+      <style>
+        /* Example styles. Added to make the action buttons responsive. */
+        .actions-container {
+          display: flex;
+          gap: 16px;
+          flex-wrap: wrap;
+          @media (max-width: 42rem) {
+            kd-button {
+              width: 100%;
+              flex-grow: 1;
+            }
+          }
+        }
+      </style>
+      ${story()}
+    `,
+  ],
+  render: (args) => {
+    return html`
+      <kyn-error-block titleText=${args.titleText}>
+        <!-- <div slot="image">${unsafeHTML(alertImg)}</div> -->
+        <!-- <div slot="image">${unsafeHTML(notFoundImg)}</div> -->
+        <!-- <div slot="image">${unsafeHTML(stopImg)}</div> -->
+        <div slot="image">${unsafeHTML(timeoutImg)}</div>
```

Comment:
Use `unsafeSVG` instead of `unsafeHTML`.

Reply:
Sure, but the svg missing part issue is still there